.github/CODEOWNERS

```diff
@@ -254,7 +254,7 @@ app/models/form1010cg/submission.rb @department-of-veterans-affairs/vfs-10-10 @d
 app/models/form1095_b.rb @department-of-veterans-affairs/vfs-1095-b @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/models/form526_job_status.rb @department-of-veterans-affairs/Disability-Experience @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/models/form526_submission.rb @department-of-veterans-affairs/Disability-Experience @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-app/models/form_attachment.rb @department-of-veterans-affairs/Benefits-Team-1 @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
+app/models/form_attachment.rb @department-of-veterans-affairs/benefits-decision-reviews-be @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
```

Comment:
@shaunburdick @dfong-adh Not sure if we should be the owners on this one... Seems like other teams would probably touch it, but it was assigned to `Benefits-Team-1` so 🤷‍♀️ I can change it back if it should be replaced with a broader team

Reply:
We can take it for now and see who ends up working on it when they open a PR.